kernel/kernel-util/src/main/java/org/sakaiproject/util/EmailNotification.java

```diff
@@ -645,6 +645,24 @@ protected int getOption(User user, String notificationId, String resourceFilter,
 		String type = getType(resourceFilter);
 		if (type != null)
 		{
+            // First, check the overrides.
```

Comment:
I will be more than happy to replace the hard tabs in this file with soft tabs of four spaces. I hate hard tabs, they are just a pain in the derriere. I'll do that in another pull request.